BDArmory/Control/BDModuleOrbitalAI.cs

```diff
@@ -1451,11 +1556,89 @@ public static float GetMaxThrust(Vessel v)
             return thrust;
         }
 
-        private static float GetConsumptionRate(Vessel v)
+        private void UpdateEngineLists(bool forceUpdate = false)
+        {
+            // Update lists of engines that can provide forward and reverse thrust
+            if (!ReverseThrust && !forceUpdate) return;
+            forwardEngines.Clear();
+            reverseEngines.Clear();
+            float forwardThrust = 0f;
+            float reverseThrust = 0f;
+            foreach (var engine in VesselModuleRegistry.GetModuleEngines(vessel))
```

Comment:
This won't catch FireSpitter engines, but that's probably not an issue for spacecraft.